types/backbone/index.d.ts

```diff
@@ -64,14 +68,10 @@ declare namespace Backbone {
         parse?: boolean;
     }
 
-    interface PersistenceOptions {
-        url?: string;
-        data?: any;
-        beforeSend?: (jqxhr: JQueryXHR) => void;
-        timeout?: number;
-        // TODO: copy all parameters from JQueryAjaxSettings except success/error callbacks?
-        success?: (modelOrCollection?: any, response?: any, options?: any) => void;
-        error?: (modelOrCollection?: any, jqxhr?: JQueryXHR, options?: any) => void;
+    interface PersistenceOptions extends Partial<_Omit<JQueryAjaxSettings, 'success' | 'error'>> {
```

Comment:
Just to confirm, `PersistenceOptions` and `JQueryAjaxSettings` are inherently related?  If new properties are added to `JQueryAjaxSettings`, they should appear here as well?  Otherwise, this seems like an unnecessarily complicated way to eliminate a small amount of code duplication.

Reply:
Yes, these options are actually forwarded to jQuery.

Reply:
Thanks for confirming!